docs/pages/all.html

```diff
@@ -8,7 +8,7 @@
 
 <p>Welcome to the alpha site for the Draft U.S. Web Design Standards project.</p>
 
-<p>This Draft Web Design Standards is intended to provide reference for designers, developers, as well as anyone else interested in the design and frontend principles that make up the building blocks of federal government websites.</p>
+<p>The Draft U.S. Web Design Standards is intended to provide reference for designers, developers, as well as anyone else interested in the design and frontend principles that make up the building blocks of federal government websites.</p>
```

Comment:
A few notes:

1) Please change "Standards is" to "Standards are" (unless "Standards" is considered a singular item). 
2) I'd change "designers, developers, as well as anyone" to "designers, developers, and anyone else interested ..."
3) Please update "frontend" to "front end"